workspaces/liferay-customer-workspace/client-extensions/liferay-customer-etc-spring-boot/src/main/java/com/liferay/customer/constants/HeatScoreConstants.java

```diff
@@ -0,0 +1,126 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.customer.constants;
+
+import com.liferay.petra.string.StringPool;
+
+/**
+ * @author Jenny Chen
+ */
+public interface HeatScoreConstants {
+
+	public static final String EVENT_TYPE_GO_LIVE = "goLive";
+
+	public static final String EVENT_TYPE_MIGRATION = "migration";
+
+	public static final String EVENT_TYPE_OTHER_EVENT = "otherEvent";
+
+	public static final String EVENT_TYPE_UPGRADE = "upgrade";
+
+	public static final String HEAT_TAG_CUSTOMER_UPGRADE = "customer_upgrade";
```

Comment:
Can remove the HEAT_TAG prefix if renaming to HeatTagConstants and removing eventTypes